package.json

```diff
@@ -64,11 +63,16 @@
     "eslint": "3.6.0",
     "font-awesome": "4.6.3",
     "handlebars": "4.0.5",
+    "imports-loader": "0.6.5",
     "istanbul": "0.4.5",
+    "jquery": "2.1.1",
+    "jquery-ui": "1.12.1",
```

Comment:
It did work. I wasn't able to consistently reproduce #494 on either old or new jquery-ui, but i did get it to happen once or twice so probably isn't fixed.